app/src/main/kotlin/com/wire/android/notification/WireNotificationManager.kt

```diff
@@ -399,7 +400,7 @@ class WireNotificationManager @Inject constructor(
     private suspend fun observeOngoingCalls(currentScreenState: StateFlow<CurrentScreen>) {
         currentScreenState
             .flatMapLatest { currentScreen ->
-                if (currentScreen !is CurrentScreen.InBackground) {
+                if (Build.VERSION.SDK_INT < Build.VERSION_CODES.UPSIDE_DOWN_CAKE && currentScreen !is CurrentScreen.InBackground) {
```

Comment:
This means that for versions starting with `UPSIDE_DOWN_CAKE` it can start ongoing call service regardless of whether the app is in the background or foreground, but for older versions only when the app is in the background - is this intended?

Reply:
yes